home/templates/layouts/base.html

```diff
@@ -67,69 +65,114 @@
-  
-  {% endblock header %}
 
-  {% block content %}{% endblock content %}
+    {% endblock header %}
+
+    {% block content %}{% endblock content %}
+
+    {% block footer %}
 
-  {% block footer %}
-  
     {% include 'includes/footer.html' %}
-  
-  {% endblock footer %}
 
-  <button id="backToTop" style="
+    {% endblock footer %}
+
+    <button id="backToTop" style="
   position: fixed;
   bottom: 50px;
-  left: 95%;
+  left: 98.5%;
   transform: translateX(-50%);
   background-color: #3a3a32;
-  color: rgb(236, 232, 2); 
-  padding: 5px; 
+  color: rgb(236, 232, 2);
+  padding: 5px;
   border: none;
   border-radius: 30%;
   cursor: pointer;
   z-index: 1000;
   display: none;
-  width: 35px; 
-  height: 35px; 
+  width: 35px;
+  height: 35px;
   display: flex;
   opacity: 0;
   align-items: center;
   justify-content: center;
   font-size: 15px;
-  transition: opacity 0.5s ease-in-out;"> 
-  &#8593;
-</button>
-
-  {% include 'includes/scripts.html' %}
-  {% block javascripts %}{% endblock javascripts %}
-<!--Subresource Integrity-->
-<script type="module"
-src="https://unpkg.com/ionicons@7.1.0/dist/ionicons/ionicons.esm.js" 
-integrity="sha384-xyQ/Vpt1cq1pdPYlRaT/QkcSj2vrYyWI3WOhxGBzqnP1vgvsS2yrhWXe3/wIoCr5" 
-crossorigin="anonymous"></script>
-
-<script type = "module"
-src="https://unpkg.com/ionicons@7.1.0/dist/ionicons/ionicons.js" 
-integrity="sha384-KAUcUASKfGZ0sxENVFRiH12jtpvd9SsVpgzvYdvv8ZMLqB99G3gX3DAts418wzIC" 
-crossorigin="anonymous"></script>
-<!--Internal script no SRI needed-->
-<script src="{% static 'backToTop.js' %}"></script>
+  transition: opacity 0.5s ease-in-out;">
+        &#8593;
+    </button>
+
+    <!-- Floating Call Button -->
+    <div id="callButton">
+        📞
+    </div>
+
+    <!-- Overlay for the popup -->
+    <div id="overlay">
+        <div class="col-md-8 card form-card-ov">
+            <h2>Contact Us</h2>
+            <form id="contactForm">
+                <div class="mb-3">
+                    <label for="name" class="form-label">Your Name</label>
+                    <input type="text" id="name" name="name" class="form-control" placeholder="Enter your name" required>
+                </div>
+                <div class="mb-3">
+                    <label for="email" class="form-label">Your Email</label>
+                    <input type="email" id="email" name="email" class="form-control" placeholder="Enter your email" required>
+                </div>
+                <div class="mb-3">
+                    <label for="message" class="form-label">Your Message</label>
+                    <textarea id="message" name="message" class="form-control" rows="4" placeholder="Enter your message" required></textarea>
+                </div>
+                <button type="submit" id="submitForm" class="btn btn-success w-100">Submit</button>
+                <button onclick="closePopup()" id="closeForm" class="btn btn-secondary w-100 mt-3">Close</button>
+            </form>
+        </div>
+    </div>

```

Comment:
currently this has the form submit as a get request back to the same page, it should instead be sent as a post request to the main contact form